teuthology/nuke.py

```diff
@@ -258,9 +258,9 @@ def remove_installed_packages(ctx):
     install_task.remove_packages(
         ctx,
         conf,
-        {"deb": install_task.deb_packages['ceph'] +
+        {"deb": install_task.PACKAGES['ceph']['deb'] +
          ['salt-common', 'salt-minion', 'calamari-server'],
```

Comment:
Can these be part of PACKAGES as well, seems weird to tack them on here.  Maybe a common_packages list of some sort that gets added to both the deb and rpm lists?